### ISSUE TITLE
Bless losing item.

### ISSUE THREAD

**Issue by pedrogiampietro** [malucooo/otxserver-new#96]
is already the tenth player that complains that it loses some type of item of the body when using the bless, I even checked, with a char, I bought the bless, I used the ankh to see if it had bought all, and in a rarities I lost 1 item of the body.

https://i.imgur.com/sk61G9f.jpg

**Comment by favasconcelos** [comment]
Have you made any changes to data/creaturescripts/scripts/others/droploot.lua ?

**Comment by favasconcelos** [comment]
If it's possible, can you provide more information about how the players died ?

- Killed by players, monsters, summons ?
- Any of the killers had any summons ?
- Did they have any sort of skulls ?

**Comment by pedrogiampietro** [comment]
first there were players reporting that they were losing some of the equipment for me, specifically the boot.
so I tested a few times and did not lose anything, and said he was forgetting to buy bless, but how come? if he does not lose the backpack but loses an item from the body? (I.e.

so I came back to test again and after several deaths I was able to drop 1 item of same blessing, dying for monsters
